experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/R.kt

```diff
@@ -18,21 +18,22 @@ object ResString {
     val refresh: String
 
     init {
-        if (System.getProperty("user.language").equals("ru")) {
+        println(System.getProperty("user.language"))
+        if (System.getProperty("user.language").equals("de")) {
             appName = "ImageViewer"
-            loading = "Загружаем изображения..."
-            repoEmpty = "Репозиторий пуст."
-            noInternet = "Нет доступа в интернет."
-            repoInvalid = "Список изображений в репозитории пуст или имеет неверный формат."
-            refreshUnavailable = "Невозможно обновить изображения."
-            loadImageUnavailable = "Невозможно загузить полное изображение."
-            lastImage = "Это последнее изображение."
-            firstImage = "Это первое изображение."
-            picture = "Изображение:"
-            size = "Размеры:"
-            pixels = "пикселей."
-            back = "Назад"
-            refresh = "Обновить"
+            loading = "Bilder werden geladen..."
+            repoEmpty = "Bildverzeichnis ist leer."
+            noInternet = "Kein Internetzugriff."
```

Comment:
Yeah, it's a good idea to add Deutsch locale.
Also, we have a locale here for Android:
shared/src/androidMain/res/values-ru/strings.xml

Reply:
I've replaced it with a `values-de` XML!
Do we have any option to easily prevent the duplication between locale specification across platforms? 🤔

Reply:
For now — not.
But, another one library has added parser of an Android strings format for all platforms (https://github.com/icerockdev/moko-resources#example-1---simple-localization-string)
We can do the same in the future.

Reply:
>Yeah, it's a good idea to add Deutsch locale.

But I am not sure if it is ok to remove Russian locale. Maybe we may just leave English until we have a good Multiplatform solution for localized Strings?

Reply:
> Multiplatform solution for localized Strings?

We use moko-resources or define strings in [code itself](https://github.com/JetBrains/compose-jb/issues/425#issuecomment-791305807)